ios/ReactNativeBase/Info.plist

```diff
@@ -37,8 +37,6 @@
 	</dict>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string></string>
-	<key>UILaunchStoryboardName</key>
-	<string>LaunchScreen</string>
```

Comment:
is this ok? 🤔

Reply:
I'm not sure actually, it seems like it was an unintentional change. I will revert it and re-test it

Reply:
It was automatically moved some lines below

Reply:
Good catch!